src/main/webapp/resources/js/contexts/PagedTableContext.js

```diff
@@ -66,14 +66,20 @@ function PagedTableProvider({ children, url }) {
   const handleTableChange = (pagination, filters, sorter) => {
     const { pageSize, current } = pagination;
     const { order, field } = sorter;
+
+    // Need to remove null filters in ant design v4
```

Comment:
Is this used by say for example "States" is null then it won't display the filter icon for States?

Reply:
No ant v4 just sends null (previously nothing) which messes with our server code.  So if they are null we remove them client side before sending to server.